AeroGearOAuth2/OAuth2Module.swift

```diff
@@ -133,7 +134,12 @@ open class OAuth2Module: AuthzModule {
                 self.webView!.targetURL = url
                 config.webViewHandler(self.webView!, completionHandler)
             } else {
-                UIApplication.shared.openURL(url)
+                if #available(iOS 9.0, *) {
```

Comment:
@corinnekrych This is missing a vital piece: guaranteeing that use of GoogleConfig always uses the external browser or `SFSafariViewController`. My initial thought is in the if clause, to do something like `if self.webView != nil && type(of: config) != GoogleConfig.self` but that seems a bit hacky. 

In any case, the change should also be reflected in the documentation.